test/CMakeLists.txt

```diff
@@ -26,9 +26,15 @@ FIND_PACKAGE(alpaka REQUIRED)
 
 ADD_SUBDIRECTORY("common/")
 
-find_package(Catch2 2.4.0 CONFIG REQUIRED)
-set_target_properties(Catch2::Catch2 PROPERTIES IMPORTED_GLOBAL TRUE)
-message(STATUS "Catch2: Found version ${Catch2_VERSION}")
+OPTION(ALPAKA_USE_INTERNAL_CATCH "Use internally shipped Catch2" ON)
+
+IF(ALPAKA_USE_INTERNAL_CATCH)
```

Comment:
Could we rename it to `ALPAKA_USE_INTERNAL_CATCH2`?

Reply:
sure!